shared_crates/schema/src/schema/physics.toml

```diff
@@ -157,5 +157,7 @@ attributes = ["Debuggable", "Networked", "Store"]
 [components."core::physics::visualize_collider"]
 type = "Empty"
 name = "Visualize collider"
-description = "If attached, the collider will be rendered."
+description = """If attached, the collider will be rendered.
+**Note**: this will hijack `local_gizmos`
```

Comment:
Clarify what this means - does it replace it entirely or add to it?

Reply:
By hijack I mean a system which consistently overwrites the component, so if you store your own things there it will be lost all the time. Think writing to position while you have velocity, etc

Reply:
Mention that in the component description and we're good to go